### ISSUE TITLE
Creating Configuration file (.YAML) for new dataset

### ISSUE THREAD

**Issue by AnishaKarthyedath** [pyannote/pyannote-audio#710]
Hi,

The tutorial given for pyannote says that experimental protocol usually contains three subsets train, development, and test. I am using a small dataset for speaker diarization. I need to Fine-tune a pre-trained model using my dataset. When I create the database.yaml file for fine-tuning ,is it necessary to include the three subsets to the configuration file or can I just define the entire dataset as the training dataset. Please correct me if I am wrong in the concept. How can I create the database .yaml file?

example file:

Databases:
   abc: ./path to/*/audio/{uri}.wav
Protocols:
   abc:
      SpeakerDiarization:
         MixHeadset:
           train:
              uri: ./filename/path.train.lst
              annotation: ./filename/path.train.rttm
              annotated: .filename/path.train.uem

**Comment by Gabrielk99** [comment]
Thats correct, in the tutorial of training model, you will see the file database.yml, just follow the same way that is in the file, like you describle here. And you dont need include all subset, well if tou protocol have only subset,

**Comment by AnishaKarthyedath** [comment]
Thank you. I used only the train subset. it worked
